src/app/teams/teams.component.ts

```diff
@@ -110,6 +110,15 @@ export class TeamsComponent implements OnInit, AfterViewInit {
     });
   }
 
+  deleteTeam(team) {
+    this.teamsService.deleteTeam(team).subscribe((newTeam: any) => {
+      this.planetMessageService.showMessage('You have deleted a team.');
+      if (newTeam.status === 'archived') {
+        this.teams.data = this.teams.data.filter((t: any) => t.doc._id !== newTeam._id);
```

Comment:
Please refactor to create a new method for this line which is used here and in `toggleMembership`.  Let's call the new method `removeTeamFromTable`